packages/megacmd/build.sh

```diff
@@ -0,0 +1,32 @@
+TERMUX_PKG_HOMEPAGE=https://mega.nz/
+TERMUX_PKG_DESCRIPTION="Provides non UI access to MEGA services"
+TERMUX_PKG_LICENSE="BSD 2-Clause"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=1.4.1
+_COMMIT=fd50adec27d1c2fdea8f7264cb30b8d8117d0699
+TERMUX_PKG_SRCURL=https://github.com/meganz/MEGAcmd.git
+TERMUX_PKG_GIT_BRANCH=master
+_SDK_COMMIT=0e79b2739f695d08efed5a61bbf44362e127c30b
+TERMUX_PKG_DEPENDS="c-ares, cryptopp, ffmpeg, freeimage, libc++, libcurl, libsodium, libsqlite, libuv, libzen, mediainfo, openssl, pcre, readline, zlib"
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_RM_AFTER_INSTALL="lib/libpthread.a"
```

Comment:
This will remove file only from package but leave it in `$TERMUX_PREFIX`. Please remove file using `termux_step_post_make_install` build step.

Reply:
Fixed. Thanks!